packages/scss/src/theming/components/_list.theme.scss

```diff
@@ -1,5 +1,8 @@
 $list: (
   margin-bottom: 1rem,
   padding: 1rem 1.5rem,
+  hover: (
+    background: #EEEEEE
```

Comment:
I wonder if we shouldn't add in the $colors map a "background" map including default white and variations of greys

Reply:
I was thinking to use a common.hovering.background 🤔

Reply:
Will be done in another PR